pipeline/console/container/__init__.py

```diff
@@ -260,6 +260,26 @@ def _build_container(namespace: Namespace):
     #     _print(f"Created tag {pipeline_repo}:{pipeline_tag}", "SUCCESS")
 
 
+def _check_user_has_run_container_up():
+    user_input = (
+        input(
+            "Have you checked your pipeline works locally by running 'pipeline container up'? (yes/no): "
+        )
+        .strip()
+        .lower()
+    )
+    if user_input == "yes":
+        return
+    elif user_input == "no":
+        _print(
+            "Please run the command 'pipeline container up' before continuing. This will help ensure your pipeline works as expected before uploading it to Mystic."
```

Comment:
Think we should maybe say something like "We recommend trying `pipeline container up`", since some people won't have the GPU hardware to do it and may think its a requirement.